app/test/std_sdk_test.go

```diff
@@ -241,51 +251,52 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 				msgSend1 := banktypes.NewMsgSend(
 					testfactory.GetAddress(s.cctx.Keyring, account1),
 					testfactory.GetAddress(s.cctx.Keyring, account2),
-					sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1))),
+					sdk.NewCoins(sdk.NewCoin(params.BondDenom, math.NewInt(1))),
 				)
 				account3 := s.unusedAccount()
 				msgSend2 := banktypes.NewMsgSend(
 					testfactory.GetAddress(s.cctx.Keyring, account1),
 					testfactory.GetAddress(s.cctx.Keyring, account3),
-					sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1))),
+					sdk.NewCoins(sdk.NewCoin(params.BondDenom, math.NewInt(1))),
 				)
 				return []sdk.Msg{msgSend1, msgSend2}, account1
 			},
 			expectedCode: abci.CodeTypeOK,
 		},
-		{
-			name: "create param change proposal for a blocked parameter",
-			msgFunc: func() (msgs []sdk.Msg, signer string) {
-				account := s.unusedAccount()
-				change := proposal.NewParamChange(stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom), "stake")
-				content := proposal.NewParameterChangeProposal("title", "description", []proposal.ParamChange{change})
-				addr := testfactory.GetAddress(s.cctx.Keyring, account)
-				msg, err := oldgov.NewMsgSubmitProposal(
-					content,
-					sdk.NewCoins(
-						sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000000))),
-					addr,
-				)
-				require.NoError(t, err)
-				return []sdk.Msg{msg}, account
-			},
-			// this parameter is protected by the paramfilter module, and we
-			// should expect an error. Due to how errors are bubbled up, we get
-			// this code despite wrapping the expected error,
-			// paramfilter.ErrBlockedParameter
-			expectedCode: govtypes.ErrNoProposalHandlerExists.ABCICode(),
-		},
+		// TODO: paramfilter module is removed, should be replaced by ante handler
+		// {
+		// 	name: "create param change proposal for a blocked parameter",
+		// 	msgFunc: func() (msgs []sdk.Msg, signer string) {
+		// 		account := s.unusedAccount()
+		// 		change := proposal.NewParamChange(stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom), "stake")
+		// 		content := proposal.NewParameterChangeProposal("title", "description", []proposal.ParamChange{change})
+		// 		addr := testfactory.GetAddress(s.cctx.Keyring, account)
+		// 		msg, err := oldgov.NewMsgSubmitProposal(
+		// 			content,
+		// 			sdk.NewCoins(
+		// 				sdk.NewCoin(params.BondDenom, math.NewInt(1000000000))),
+		// 			addr,
+		// 		)
+		// 		require.NoError(t, err)
+		// 		return []sdk.Msg{msg}, account
+		// 	},
+		// 	// this parameter is protected by the paramfilter module, and we
+		// 	// should expect an error. Due to how errors are bubbled up, we get
+		// 	// this code despite wrapping the expected error,
+		// 	// paramfilter.ErrBlockedParameter
+		// 	expectedCode: govtypes.ErrNoProposalHandlerExists.ABCICode(),
+		// },
```

Comment:
you probably already were, but we should have issues for all followups before merging, or ofc if no flup in needed then delete this.

Reply:
absolutely, we've been tracking issues through linear. most things should be accounted for with follow up issues or open questions for us to discuss together, but anything you see please do point out in the PR and we can address

Reply:
addressed here https://github.com/01builders/celestia-app/pull/62, should be merged soon and reflected here